reactjs-todo/package.json

```diff
@@ -38,7 +38,7 @@
     "webpack-cli": "^5.1.4"
   },
   "dependencies": {
-    "@forgerock/javascript-sdk": "^4.2.0",
+    "@forgerock/javascript-sdk": "^4.4.0",
```

Comment:
Can we just make this a `"*"` so that we are always looking for whatever latest version of the sdk is?

That was my thought last week, and would reduce our need to update the versions here